Remove stale file references from WebCore.xcodeproj

emw-apple opened the pull request:
#### 68de618b1319e373da99727d3faaddebe9486cfa
<pre>
Remove stale file references from WebCore.xcodeproj
<a href="https://bugs.webkit.org/show_bug.cgi?id=274971">https://bugs.webkit.org/show_bug.cgi?id=274971</a>
<a href="https://rdar.apple.com/problem/129401444">rdar://problem/129401444</a>

Reviewed by Alexey Proskuryakov.

Clean up a bunch of file references for things that have been deleted in
the following commits:

<a href="https://commits.webkit.org/214790@main">https://commits.webkit.org/214790@main</a>
<a href="https://commits.webkit.org/229645@main">https://commits.webkit.org/229645@main</a>
<a href="https://commits.webkit.org/231818@main">https://commits.webkit.org/231818@main</a>
<a href="https://commits.webkit.org/240102@main">https://commits.webkit.org/240102@main</a>
<a href="https://commits.webkit.org/255702@main">https://commits.webkit.org/255702@main</a>
<a href="https://commits.webkit.org/268163@main">https://commits.webkit.org/268163@main</a>
<a href="https://commits.webkit.org/268768@main">https://commits.webkit.org/268768@main</a>
<a href="https://commits.webkit.org/268768@main">https://commits.webkit.org/268768@main</a>
<a href="https://commits.webkit.org/272998@main">https://commits.webkit.org/272998@main</a>
<a href="https://commits.webkit.org/276115@main">https://commits.webkit.org/276115@main</a>
<a href="https://commits.webkit.org/276683@main">https://commits.webkit.org/276683@main</a>
<a href="https://commits.webkit.org/277560@main">https://commits.webkit.org/277560@main</a>

For CryptoKitShim.swift and JSInternalReadableStream.h, these files
appear to have never been in use -- they were cruft when they were
originally checked in.

Canonical link: <a href="https://commits.webkit.org/280353@main">https://commits.webkit.org/280353@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7456d0d2add88a1fb943001bcee9892fe28017ec

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/54954 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/34407 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/7543 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/58232 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/5685 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/57254 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/41957 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/5717 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/44502 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/3859 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/57049 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/32493 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/47606 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/25628 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/29282 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/4949 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/3826 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/51137 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/5173 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/59823 "Built successfully") | 
| | [⏳ 🛠 vision ](https://ews-build.webkit.org/#/builders/visionOS-1-Build-EWS "Waiting in queue, processing has not started yet") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/5326 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/51922 "Passed tests") | 
| | [⏳ 🛠 vision-sim ](https://ews-build.webkit.org/#/builders/visionOS-1-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/47684 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/51360 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/12482 "Built successfully and passed tests") | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-1-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/30218 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/31351 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/32366 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/31139 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov reviewed (approved):
It would be good to verify that internal builds are passing.

emw-apple committed:
Remove stale file references from WebCore.xcodeproj
https://bugs.webkit.org/show_bug.cgi?id=274971
rdar://problem/129401444

Reviewed by Alexey Proskuryakov.

Clean up a bunch of file references for things that have been deleted in
the following commits:

https://commits.webkit.org/214790@main
https://commits.webkit.org/229645@main
https://commits.webkit.org/231818@main
https://commits.webkit.org/240102@main
https://commits.webkit.org/255702@main
https://commits.webkit.org/268163@main
https://commits.webkit.org/268768@main
https://commits.webkit.org/268768@main
https://commits.webkit.org/272998@main
https://commits.webkit.org/276115@main
https://commits.webkit.org/276683@main
https://commits.webkit.org/277560@main

For CryptoKitShim.swift and JSInternalReadableStream.h, these files
appear to have never been in use -- they were cruft when they were
originally checked in.

Canonical link: https://commits.webkit.org/280353@main
